### ISSUE TITLE
Accessibility Chapter questions over Figure 1

### ISSUE THREAD

**Issue by bazzadp** [HTTPArchive/almanac.httparchive.org#856]
Pulling this out from #854 raised by @juliemoynat

[Figure 1 in the Accessibility chapter](https://almanac.httparchive.org/en/2019/accessibility#fig-1) is the following:

![image](https://user-images.githubusercontent.com/10931297/82550805-049bc980-9b57-11ea-98c0-0f1dfb9aa504.png)

And has the following in the (hidden) description (emphasis added by myself to one sentence):

> Four colored boxes of orange and gray shades with white text overlaid inside creating two columns. The left column says Too lightly colored and has the orange background color written as #FCA469. The right column says Recommended and the orange background color is written as #F56905. The top box in each column has an orange background with white text #FFFFFF and the bottom box has a gray background with white text #FFFFFF. **The grey background shows the orange color in greyscale**. Courtesy of LookZook.

It is not clear from the image nor the text that "The grey background shows the orange color in greyscale" so if that is the intention, then it should be made clear. If that is not the intention then the long description should be corrected.

**Comment by bazzadp** [comment]
See also #855 where there is a suggestion to replace this image, and perhaps make this more clear with a heading above the greyscale images. Still think it warrants adding something to the text though in addition to that.
